roles/ceph-osd/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: install dependencies
+  apt: pkg=parted
+       state=present
+
+- name: write out bootstrap keyring
+  copy:
+    dest: "{{ hostvars[groups['mons'][0]]['keyrings']['source'] }}"
+    content: "{{ hostvars[groups['mons'][0]]['keyrings']['content'] | b64decode }}"
+
+- name: prepare osd disk(s)
+  command: "ceph-disk prepare {{ item.0 }} {{ item.1 }}"
+  with_together:
+    - ceph_osd.devices
+    - ceph_osd.raw_journal_devices
+  changed_when: false
+  failed_when: false
+
+- name: activate osd(s)
+  command: ceph-disk activate {{ item | regex_replace('^(\/dev\/cciss\/c[0-9]{1}d[0-9]{1})$', '\\1p') }}1
+  with_items: ceph_osd.devices
+  failed_when: false
```

Comment:
Hmm guess I don't have enough experience with `changed_when` and `failed_when`. Take them out?